controllers/details.js

```diff
@@ -1,20 +1,19 @@
 'use strict';
 
 var _ = require('underscore');
-var Model = require('../models');
-var helpers = require('../lib/helpers');
-
-module.exports = function renderDetails(req, res) {
-  var model = new Model(req.session.model);
-  var record = {record: model.get('records')[0]};
+var api = require('../api');
 
+module.exports = function renderDetails(req, res, next) {
   if (req.params && req.params.sysnum) {
-    record.record = _.findWhere(model.toJSON().records, {
-      'system-number': parseInt(req.params.sysnum, 10)
-    });
+    api.read({
+      'system-number': req.params.sysnum
+    }).then(function resolved(result) {
+        res.render('pages/details', {record: result.records[0]});
```

Comment:
If the API is reading based on a system number should the API return only one record, rather than us needing to get the first element here?